client/src/modules/invoices/patientInvoiceForm.service.js

```diff
@@ -465,15 +456,15 @@ Patients, PriceLists, Inventory, AppCache, Store, Pool,
     this.clear();
 
     // loop through the cached items, configuring them
-    cp.items.forEach(function (cacheItem) {
-      var item = this.addItem();
+    cp.items.forEach((cacheItem) => {
+      const item = this.addItem();
 
       item.inventory_uuid = cacheItem.inventory_uuid;
       this.configureItem(item);
 
       item.quantity = cacheItem.quantity;
       item.transaction_price = cacheItem.transaction_price;
-    }.bind(this));
```

Comment:
By removing this `.bind(this)`, the function inside the foreach can still see the parent `this` and call it like ``this.addItem(); ?

Reply:
Yup!  That is the power of arrow functions in JavaScript.  [Here is the MDN documentation on it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_separate_this)